.changeset/smooth-jokes-begin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/theme': patch
+'@backstage/plugin-catalog-graph': patch
+---
+
+Added a way to customize the nodes color on the basis of kind
```

Comment:
Would it be possible to include an example? It's really handy to have this here when doing upgrades as you don't need to jump around the code base finding things